hub/core/meta/tensor_meta.py

```diff
@@ -146,6 +137,10 @@ def _update_shape_interval(self, shape: Tuple[int, ...]):
             self.min_shape[i] = min(dim, self.min_shape[i])
             self.max_shape[i] = max(dim, self.max_shape[i])
 
+    def as_dict(self):
+        # TODO: tensor meta as_dict
```

Comment:
reminder

Reply:
not necessary right now

Reply:
i added this TODO and notimplemented because i don't want it to call the base classes to_dict, cus it won't be valid